api-utils/auth.js

```diff
@@ -1,6 +1,7 @@
 const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
 
-exports.getAuthTokens = (payload) => {
+export const getAuthTokens = (payload) => {
   // Generate Access Token
   let accessToken = jwt.sign(payload, process.env.ACCESS_TOKEN_SECRET, {
     algorithm: "HS256",
```

Comment:
create a consts file to save magic numbers and import constant values from their. You can add consts.js file in root folder

Reply:
I will add the file